index.html

```diff
@@ -55,13 +64,13 @@
 		box-shadow: 0 0 5px #e8e9eb inset;
 		width: 100%;
 		font-size:1em;
-		margin-right: 5px;
+		/* margin-right: 5px; */
 	}
 	#cc-number {
-		width: 330px !important;
+		/* width: 330px !important; */
```

Comment:
If the CSS selector has no more property in it. Don't you think its better to just remove it?